.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: java
 jdk:
-  - openjdk7
-  - oraclejdk7
-  - oraclejdk8
-install: /bin/true
-script: mvn checkstyle:check test package
+- openjdk7
+- oraclejdk7
+- oraclejdk8
```

Comment:
That must have happend when i used the `travis` commandline tool to encrypt and automatically add my token to the config. I'm gonna change that back.